### DISCUSSION TITLE
What to do with images which doesnt have anything to label?

### DISCUSSION THREAD

**Discussion by probablyanek** [ultralytics/ultralytics#3032] [Q&A]
I am working on a project to create a brain tumor identification model using MRI scans. My teacher has given me a dataset containing 1000 MRI images of brains with tumors and 1000 MRI images of tumor-free brains. While I can label and use the tumor for training, I am unsure how to handle the unlabeled tumor-free images. Should I include them in the training set without any labels?

**Comment by glenn-jocher** [comment]
@probablyanek hello! For your brain tumor identification model, it's important to include both positive (tumor) and negative (tumor-free) examples in your training set. You can indeed include the tumor-free images in the training set without any labels. In the context of object detection, these would be considered as images with no annotations, and the model will learn to recognize frames where no detections should be made. This helps the model to better discriminate between the presence and absence of tumors. Just ensure that your dataset is correctly formatted, with the tumor-free images included but with no bounding box annotations. Happy training! 😊🧠🚫
